layouts/shortcodes/code-webui-port-forward.html

```diff
@@ -0,0 +1,4 @@
+```sh
```

Comment:
This code snippet isn't rendering properly. See preview:

* https://deploy-preview-659--competent-brattain-de2d6d.netlify.com/docs/started/getting-started-minikube/#where-to-go-next
* https://deploy-preview-659--competent-brattain-de2d6d.netlify.com/docs/other-guides/accessing-uis/#using-kubectl-and-port-forwarding

I suspect you need to use HTML formatting rather than Markdown.

Reply:
Just tested locally and confirmed in will render correctly when updated with html tags.
```
<pre><code>export NAMESPACE=kubeflow
kubectl port-forward svc/ambassador -n ${NAMESPACE} 8080:80
</code></pre>
```